assets/css/components/_tecnologias.scss

```diff
@@ -7,13 +7,16 @@
       filter: none;
     }
 
-    &__imagen {
-      height: 250px;
-      object-fit: scale-down;
-    }
-
     &__titulo {
       text-align: center;
     }
   }
+
+  .card {
+    .tecnologia__imagen {
```

Comment:
I had to change the style to increase the specificity of the CSS selector and prevent that the image width was overridden by the `materialize` framework. Also, avoiding the use of the `!important` directive.